src/BalanceLaws/tendency_types.jl

```diff
@@ -94,6 +94,14 @@ corresponding to the column-vector `Yᵢ` in:
 """
 prognostic_vars(::BalanceLaw) = ()
 
+"""
+    projection(bl::BalanceLaw, ::TendencyDef, x)
+
+Provide a hook to project individual tendencies.
+Return identity by defualt
+"""
+projection(bl::BalanceLaw, ::TendencyDef, x) = x
```

Comment:
@kpamnany can you review this projection hook? It's mainly used in `src/BalanceLaws/sum_tendencies.jl`. I think that this is something that @LenkaNovak and @mwarusz may also need. To make this work on the sphere, we might want to add `args` as an argument, so that we have access to `args.aux`, which is needed for projections on the sphere.

Reply:
It looks nice, but we should probably discuss.

Reply:
@kpamnany, @simonbyrne maybe we can discuss this today sometime?